docs/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: UDS Core
+linkTitle: UDS Core
+type: docs
+menu:
+    main:
+        weight: 30
```

Comment:
```suggestion
menu:
  main:
    weight: 30
```

Reply:
Minor thing,  but I think we want to abide by our existing yaml spacing which is 2?